lib/decode.js

```diff
@@ -25,7 +25,14 @@ import util from 'util';
  */
 export async function fromCborld({
   cborldBytes, documentLoader, appContextMap, diagnose}) {
-  const cborMap = cbor.decode(cborldBytes);
+  // normalize bytes to a Buffer for 'cbor'
+  const _cborldBytes =
+    cborldBytes instanceof Buffer ? cborldBytes : Buffer.from(cborldBytes);
```

Comment:
```suggestion
    (cborldBytes instanceof Buffer) ? cborldBytes : Buffer.from(cborldBytes);
```

We don't parenthesize stuff like this?

Reply:
We haven't in the couple examples I could find.  If you find it confusing, then we should use them.  I thought it was ok.  Anyone else have an opinion?

Reply:
I don't think we need them.